doc/source/installation.rst

```diff
@@ -1,79 +1,89 @@
-.. |br| raw:: html
-
-  <br />
-
 Installation
 ============
 
-It is best to install the pycompwa framework within a virtual environment, so
-that all dependencies of pycompwa are contained within there. We recommend to
-use `Anaconda <https://www.anaconda.com/distribution/>`_ in combination with
-`Conda-Forge <https://conda-forge.org/>`_.
+Prerequisites
+^^^^^^^^^^^^^
 
-After you have `installed Anaconda
-<https://docs.anaconda.com/anaconda/install/>`_, set Conda-Forge as the
-default channel as follows:
+`pycompwa` is `available as a PyPI package
+<https://pypi.org/project/pycompwa/>`_. However, since the `pycompwa.ui` module
+contains Python bindings to the `ComPWA backend
+<https://github.com/ComPWA/ComPWA>`_, you first need to install certain C++
+prerequisites.
 
-.. code-block:: shell
+What you definitely need to install, is:
 
-  conda config --add channels conda-forge
-  conda config --set channel_priority strict
+* a build tool: `cmake <https://cmake.org/>`_
+* a C++ compiler:
+  `gcc <https://gcc.gnu.org/>`_ or `clang <https://clang.llvm.org/>`_
+* `Boost <https://www.boost.org/>`_
+* `Python3 <https://www.python.org/downloads/>`_
 
-Now, create a new environment with `all required dependencies
-<https://github.com/ComPWA/pycompwa/blob/master/requirements.txt>`_
-installed:
+The following package are optional:
 
-.. code-block:: shell
+* `ROOT <https://root.cern.ch/downloading-root>`_ and/or `Minuit2
+  <http://seal.web.cern.ch/seal/snapshot/work-packages/mathlibs/minuit/>`_
+* `GSL <https://www.gnu.org/software/gsl/>`_
+* `Geneva <https://www.gemfony.eu/>`_
+
+For basic fitting functionality, it is easiest to install ROOT `with Minuit2
+enabled <https://root.cern.ch/building-root>`_.
+
+.. hint::
 
-  conda create -n compwa --file requirements.txt
+  If you have a Linux machine with `apt <https://wiki.debian.org/Apt>`_ and
+  with administrator rights, you can run the following:
 
-Here, we call the environment ``compwa``, but you can give it any name. Now, go
-into this environment with ``conda activate compwa``, so we can install
-pycompwa there:
+  .. code-block:: shell
+
+    sudo apt update -y
+    sudo apt install -y cmake gcc git libboost-all-dev python3
+
+  ROOT with Minuit2 can be most easily installed by `downloading a suitable
+  binary for your machine <https://root.cern.ch/downloading-root>`_.
+
+
+Installation through pip
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have these dependencies installed, you can install `pycompwa` through
+`pip <https://pypi.org/project/pip/>`_. You also need to install `scikit-build
+<https://scikit-build.readthedocs.io/en/latest/>`_, because it is used as a
+build tool for the ComPWA backend:
 
 .. code-block:: shell
 
   pip install scikit-build
   pip install pycompwa
```

Comment:
Maybe add a tiny note that this way only specific releases can be installed and refer to the "interactive installation"

Reply:
https://github.com/ComPWA/pycompwa/pull/83/commits/7f8ce15fc344cfe59083bd70999229f2fb4d62e4